Plugins/Alpakit/Source/Alpakit/Private/AlpakitSettings.cpp

```diff
@@ -1,9 +1,51 @@
 #include "AlpakitSettings.h"
 
+#include "Json.h"
+
 UAlpakitSettings* UAlpakitSettings::Get() {
-    return GetMutableDefault<UAlpakitSettings>();
+	return GetMutableDefault<UAlpakitSettings>();
 }
 
 void UAlpakitSettings::SaveSettings() {
-    SaveConfig();
+	SaveConfig();
+}
+
+
+UAlpakitModSelectionSettings* UAlpakitModSelectionSettings::Get() {
```

Comment:
Can you please explain what's the point in making completely new settings just for your additions? Why can't you just reuse normal alpakit settings?